packages/dialog/src/Dialog.ts

```diff
@@ -67,23 +43,18 @@ function gatherAppliedIdsFromSlottedChildren(
  * @slot button - Button elements addressed to this slot may be placed below the content when not delivered in a fullscreen mode
  * @fires close - Announces that the dialog has been closed.
  */
-export class Dialog extends FocusVisiblePolyfillMixin(
-    ObserveSlotPresence(SpectrumElement, [
-        '[slot="hero"]',
-        '[slot="footer"]',
-        '[slot="button"]',
-    ])
-) {
+export class Dialog extends ObserveSlotPresence(AlertDialog, [
```

Comment:
Nice! Make sure we add `@spectrum-web-components/alert-dialog` to the dependencies here.

Reply:
Will add once alert-dialog is published to the npm registry.

Reply:
That's the wrong order of operations. All packages within the monorepo are dependable today, so there is no need to wait.. When we make a release with this change, were we not to include this dependency, users of the Dialog package couldn't update for the fact that the released version of Dialog would technically be broken unless they were to manually add a dependency on Alert Dialog locally.